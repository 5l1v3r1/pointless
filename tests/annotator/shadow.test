
m = 5
foo(m) = m
bar(m) = n => m