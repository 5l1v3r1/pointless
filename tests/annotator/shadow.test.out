
foo(m) = m
access: m local: 0
entry: m index: 0

bar(m) = n => m
access: m local: 1
entry: n index: 0
entry: m index: 1 upIndex: 0
entry: m index: 0

m = 5
entry: m index: 0

foo(m) = m
entry: foo index: 1

bar(m) = n => m
entry: bar index: 2
